src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -704,23 +747,69 @@ private static void addToConstantMap(ConstantSet cs, MultiMap<Class<?>, Nonrecei
   private static Set<NonreceiverTerm> constantSetToNonreceiverTerms(ConstantSet cs) {
     Set<NonreceiverTerm> result = new HashSet<>();
     for (Integer x : cs.ints) {
-      result.add(new NonreceiverTerm(JavaTypes.INT_TYPE, x));
+      result.add(new NonreceiverTerm(JavaTypes.INT_TYPE, x, cs));
     }
     for (Long x : cs.longs) {
-      result.add(new NonreceiverTerm(JavaTypes.LONG_TYPE, x));
+      result.add(new NonreceiverTerm(JavaTypes.LONG_TYPE, x, cs));
     }
     for (Float x : cs.floats) {
-      result.add(new NonreceiverTerm(JavaTypes.FLOAT_TYPE, x));
+      result.add(new NonreceiverTerm(JavaTypes.FLOAT_TYPE, x, cs));
     }
     for (Double x : cs.doubles) {
-      result.add(new NonreceiverTerm(JavaTypes.DOUBLE_TYPE, x));
+      result.add(new NonreceiverTerm(JavaTypes.DOUBLE_TYPE, x, cs));
     }
     for (String x : cs.strings) {
-      result.add(new NonreceiverTerm(JavaTypes.STRING_TYPE, x));
+      result.add(new NonreceiverTerm(JavaTypes.STRING_TYPE, x, cs));
     }
     for (Class<?> x : cs.classes) {
-      result.add(new NonreceiverTerm(JavaTypes.CLASS_TYPE, x));
+      result.add(new NonreceiverTerm(JavaTypes.CLASS_TYPE, x, cs));
     }
     return result;
   }
+
+  /**
+   * Frequency of the term in the {@link ConstantSet}
+   *
+   * @param term the literal constant
+   * @param constantSet the constant set containing the literals, frequencies, and indices.
+   * @return the frequency of the given term
+   */
+  public static int getFrequencyOfTerm(Object term, ConstantSet constantSet) {
+    Integer frequency = constantSet.constantToFrequency.get(term);
+
+    // The frequency of a term will be null if the term appears in the constant pool of the class
+    // but is never referenced in the byte code. For example, if we define a static variable,
+    // {@code public static final int mInt = 31;} that is never used, its frequency won't be
+    // defined in the map. We set the frequency to a value of 1 here to account for its appearance
+    // in the constant pool.
+    if (frequency == null) {
+      frequency = 1;
```

Comment:
We could alternatively do some sort of Laplacian smoothing (add a count of 1 to all constants mined). This might also address the case where Randoop creates a new sequence, as referenced in a previous comment.